web/bridge_types_controller.go

```diff
@@ -27,3 +31,59 @@ func (btc *BridgeTypesController) Create(c *gin.Context) {
 		c.JSON(200, bt)
 	}
 }
+
+// Index lists Bridges, one page at a time.
+
+func (btc *BridgeTypesController) Index(c *gin.Context) {
+	size, page, offset, err := ParsePaginatedRequest(c.Query("size"), c.Query("page"))
+	if err != nil {
+		c.JSON(422, gin.H{
+			"errors": []string{err.Error()},
+		})
+	}
+
+	skip := storm.Skip(offset)
+	limit := storm.Limit(size)
+
+	var bridges []models.BridgeType
+
+	if count, err := btc.App.Store.Count(&models.BridgeType{}); err != nil {
+		c.JSON(500, gin.H{
+			"errors": []string{fmt.Errorf("error getting count of Bridges: %+v", err).Error()},
+		})
+	} else if err := btc.App.Store.AllByIndex("Name", &bridges, skip, limit); err != nil {
+		c.JSON(500, gin.H{
+			"errors": []string{fmt.Errorf("erorr fetching All Bridges: %+v", err).Error()},
+		})
+	} else {
```

Comment:
We've been trying to avoid nested `if` statements. One way to do that here would be to `return` in the two error cases above, and then get rid of this else clause.